lib/libspl/Makefile.am

```diff
@@ -13,7 +13,8 @@ DEFAULT_INCLUDES += \
 	-I$(top_srcdir)/lib/libspl/include
 
 AM_CCASFLAGS = \
-	-I$(top_srcdir)/lib/libspl/include
+	-I$(top_srcdir)/lib/libspl/include \
+	$(CFLAGS)
```

Comment:
It looks like you can remove `-I$(top_srcdir)/lib/libspl/include` from `AM_CCASFLAGS` while you're here to simplify things.  `CPPASCOMPILE` and `LTCPPASCOMPILE` both already include `DEFAULT_INCLUDES`.

Reply:
Added a new commit to remove that line.